apps/codeforafrica/src/components/Footer/Footer.js

```diff
@@ -22,12 +23,24 @@ const FooterRoot = styled(Box)(
 );
 
 const Footer = React.forwardRef(function Footer(props, ref) {
-  const { subscription } = props;
+  const { subscription, menu } = props;
 
   return (
     <FooterRoot ref={ref}>
       <Section sx={{ px: { xs: "20px", sm: 0 } }}>
         <Grid container justifyContent="flex-end">
+          <Grid
+            item
+            xs={12}
+            md={3}
+            sx={{
+              order: { xs: 1, md: 0 },
+              display: "flex",
+              justifyContent: "center",
+            }}
+          >
+            <FooterMenu menu={menu} />
+          </Grid>
           <Grid item xs={12} md={3} sx={{ order: { xs: 0, md: 1 } }}>
             <NewsletterSubscription {...subscription} />
           </Grid>
```

Comment:
This order is fine but lets arrange them in mobile-first approach and use order to re-arrange them for desktop.